PendingReleaseNotes

```diff
@@ -74,38 +74,36 @@
+  config controls the maximum size to which (encoded) session metadata can grow.
 * CephFS: A new "mds last-seen" command is available for querying the last time
   an MDS was in the FSMap, subject to a pruning threshold.
-* CephFS: For clusters with multiple CephFS file systems, all the snap-schedule
+* CephFS: For clusters with multiple CephFS file systems, all snap-schedule
   commands now expect the '--fs' argument.
 * CephFS: The period specifier ``m`` now implies minutes and the period specifier
-  ``M`` now implies months. This has been made consistent with the rest
-  of the system.
+  ``M`` now implies months. This is consistent with the rest of the system.
 * RGW: New tools have been added to radosgw-admin for identifying and
   correcting issues with versioned bucket indexes. Historical bugs with the
   versioned bucket index transaction workflow made it possible for the index
   to accumulate extraneous "book-keeping" olh entries and plain placeholder
   entries. In some specific scenarios where clients made concurrent requests
-  referencing the same object key, it was likely that a lot of extra index
+  referencing the same object key, it was likely that extra index
   entries would accumulate. When a significant number of these entries are
   present in a single bucket index shard, they can cause high bucket listing
-  latencies and lifecycle processing failures. To check whether a versioned
+  latency and lifecycle processing failures. To check whether a versioned
   bucket has unnecessary olh entries, users can now run ``radosgw-admin
   bucket check olh``. If the ``--fix`` flag is used, the extra entries will
-  be safely removed. A distinct issue from the one described thus far, it is
-  also possible that some versioned buckets are maintaining extra unlinked
-  objects that are not listable from the S3/ Swift APIs. These extra objects
-  are typically a result of PUT requests that exited abnormally, in the middle
-  of a bucket index transaction - so the client would not have received a
+  be safely removed. An additional issue  is that some versioned buckets are
+  may maintain extra unlinked
+  objects that are not listable via the S3/ Swift APIs. These extra objects
+  are typically a result of PUT requests that exited abnormally in the middle
+  of a bucket index transaction, and thus the client would not have received a
   successful response. Bugs in prior releases made these unlinked objects easy
-  to reproduce with any PUT request that was made on a bucket that was actively
-  resharding. Besides the extra space that these hidden, unlinked objects
-  consume, there can be another side effect in certain scenarios, caused by
-  the nature of the failure mode that produced them, where a client of a bucket
+  to reproduce with any PUT request made on a bucket that was actively
+  resharding. In addition to the extra space that these hidden, unlinked objects
```

Comment:
It seems to me that we can dispense with "In addition to the extra space that these hidden, unlinked objects consume,". It seems to me that the position of this sentence in the text immediately after the discussion of the cost of hidden, unlinked objects already implies that the information about the object's inconsistent state is provided in addition to what has come before.